Modelica/Media/package.mo

```diff
@@ -4764,7 +4764,7 @@ are described in
       p = state.p;
       T = state.T;
       MM = MM_const;
-      R_s = 8.3144/MM;
+      R_s = Modelica.Constants.R/MM;
```

Comment:
This line looks good, it is in the interface.

Reply:
Changing it in Modelica.Media.Interfaces.PartialLinearFluid.BaseProperties also affects Modelica.Media.CompressibleLiquids.LinearColdWater and Modelica.Media.CompressibleLiquids.LinearWater_pT_Ambient via inheritance.